Smart_Currency_Converter/Smart_Currency_Converter.iOS/Info.plist

```diff
@@ -23,18 +23,20 @@
 	<key>MinimumOSVersion</key>
 	<string>8.0</string>
 	<key>CFBundleDisplayName</key>
-	<string>Smart_Currency_Converter</string>
+	<string>Smart-Currency-Converter</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.companyname.Smart_Currency_Converter</string>
+	<string>com.test.Smart-Currency-Converter</string>
 	<key>CFBundleVersion</key>
 	<string>1.0</string>
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
 	<key>CFBundleName</key>
-	<string>Smart_Currency_Converter</string>
+	<string>Smart-Currency-Converter</string>
```

Comment:
A provisioning profile is needed to run the app locally on an iOS device from Visual Studio (Visual studio for mac in my case). The provisioning profile can be created by a Xcode project and it can used by the Xamarin project if the namespace name is the the same but xcode converts dashes to hyphens, hence the use of hyphens in the naming schema above. Reference the following link for more info. https://stackoverflow.com/questions/49620026/xamarin-forms-cant-find-provisioning-profile-ios-bundle-signing

Reply:
Thanks, that make sense. The the same process is actually explained in Microsoft documentation too.

-  https://docs.microsoft.com/en-us/xamarin/ios/get-started/installation/device-provisioning/free-provisioning?tabs=windows
- https://docs.microsoft.com/en-us/xamarin/ios/get-started/installation/device-provisioning/

However since for publishing the app an Apple provisioning membership needs to be purchased, I'd be better just to proceed with that and use that provisioning profile for local testing too.